src/material-experimental/mdc-dialog/dialog.spec.ts

```diff
@@ -1670,6 +1670,17 @@ describe('MDC-based MatDialog', () => {
           .withContext('Expected the aria-labelledby to match the title id.')
           .toBe(title.id);
       }));
+
+      it('should add mat-mdc-dialog-actions-align-* class according to given [align] input in [mat-dialog-actions]', () => {
```

Comment:
Change to `it('should add correct css class according to given [align] input in [mat-dialog-actions]', () => {`